Newton-Gripper/Newton-Gripper.ino

```diff
@@ -263,29 +283,10 @@ float readVoltage() {
  *
  * Calculates stall current for given velocity and voltage (Amperes)
  ******************************************************************************/
-float stallCurrent(float velocity, float voltage) {
-  return (I_STALL_A0 + I_STALL_A1*voltage)*exp(I_STALL_B1*abs(velocity));
+float stallCurrent(float voltage) {
+  return constrain((voltage/(R_MOT*MOT_FS)),0,MOT_I_MAX);
```

Comment:
can you please explain what is happening before/after and the reason for the change?

Reply:
I removed the velocity from the stall current calculation to simplify the calculation. I think this was used compensate for getting stuck on the endstops. The new calculation makes more sense physically by using the resistance of the motor to calculate the stall current at different voltages. The constrain function is used to set a max current below the calculated stall current at higher voltages.